### PR TITLE
PP-1241: pbs_habitat fails when psql -V command has multi-line output

### PR DESCRIPTION
**Pull request by sid2364**
#### Bug/feature Description
* *Bugs: pbs_habitat does not handle the case where the output of psql -V command is multi-line*
* *[PP-1241](https://pbspro.atlassian.net/browse/PP-1241)*

#### Affected Platform(s)
* *Linux*

#### Cause / Analysis / Design
* *While parsing output of psql -V command, we don't consider a multi-line output*

#### Solution Description
* *Changed the way the output of the psql -V command is captured in the shell variable for comparison to look at only the first line (which contains the version number we're looking for)*

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [x] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by sid2364** [comment]
[psql_V_reproduction_and_unit_testing.txt](https://github.com/PBSPro/pbspro/files/1774727/psql_V_reproduction_and_unit_testing.txt)

**Comment by sid2364** [comment]
@PBSPro/pbspro-maintainers, please have a look at this pull request.

File: src/cmds/scripts/pbs_habitat.in [hunk 1]
```diff
@@ -240,7 +240,7 @@ upgrade_pbs_database() {
 		return 1
 	fi
 	
-	sys_pgsql_ver=$(echo `${PGSQL_DIR}/bin/psql -V` | awk '{print $NF}' | cut -d '.' -f 1,2)
+	sys_pgsql_ver=$(echo `${PGSQL_DIR}/bin/psql -V` | awk 'NR==1' | awk '{print $NF}' | cut -d '.' -f 1,2)
```
Comment (mike0042) [inline src/cmds/scripts/pbs_habitat.in]: Why not just add NR==1 to the existing awk call?

**Comment by sid2364** [comment]
Thanks for reviewing this, @mike0042 and @nithinj. I've squashed my commits and rebased.

**Comment by mike0042** [comment]
Merged another PR before this one. Please rebase once again @sid2364

**Comment by mike0042** [comment]
And please sign your commit. Travis is failing as a result.

**Comment by sid2364** [comment]
@mike0042, rebased it again.

**Pull request merged**